### ISSUE TITLE
Use React with snowpack instead of webpack

### ISSUE THREAD

**Issue by trivikr** [aws-samples/aws-sdk-js-v3-workshop#231]
**Is your feature request related to a problem? Please describe.**
CRA is way to bloated. The commands take time to run. It also has too many dependencies, some of them get security warnings from time-to-time. 
Here's an article from Feb 2019 which goes through CRA dependencies in detail: https://hackernoon.com/create-react-app-is-way-too-bloated-5db07c3511

**Describe the solution you'd like**
Use a faster frontend build tool like [Snowpack](https://www.snowpack.dev/).
CreateApp frontend build config generator just added support for Snowpack at: https://createapp.dev/snowpack

**Describe alternatives you've considered**
Stick with create-react-app. It's slow, but stable and gets work done.

**Comment by trivikr** [comment]
Create Snowpack App is an option to replace CRA with https://github.com/snowpackjs/snowpack/tree/main/create-snowpack-app

**Comment by trivikr** [comment]
The streaming imports in Snowpack 3.0 look interesting!
The dependencies are fetched behind the scenes, are pre-built and ready to run in the browser.

The release candidate blog post https://www.snowpack.dev/posts/2020-12-03-snowpack-3-release-candidate
Snowpack v3.0 will release on January 6th, 2021

**Comment by trivikr** [comment]
Snowpack 3.0 with streaming npm imports is releasing [on January 12th, 2021](https://twitter.com/FredKSchott/status/1348912436489818113)

**Comment by trivikr** [comment]
Snowpack 3.0 is released https://www.snowpack.dev/posts/2021-01-13-snowpack-3-0

**Comment by trivikr** [comment]
WIP commits were pushed to `react-snowpack-app` branch https://github.com/trivikr/aws-sdk-js-v3-workshop/commits/react-snowpack-app

Currently blocked on JSON named imports being discussed in https://github.com/snowpackjs/snowpack/discussions/2276

**Comment by trivikr** [comment]
The react-snowpack-app build fails with the following error in v2:
```
TypeError: Cannot read property 'memoizedProperty' of undefined
```

Experimental commits https://github.com/trivikr/aws-sdk-js-v3-workshop/commits/react-snowpack-app

**Comment by trivikr** [comment]
Closing in favor of implementation in ToDo App in modular SDK for JavaScript in https://github.com/aws-samples/aws-sdk-js-todo-app/issues/2
